### DISCUSSION TITLE
Great & Simple RTSP Server

### DISCUSSION THREAD

**Discussion by ceewanna** [bluenviron/mediamtx#1182] [General]
Just want to express my appreciation for your implementation. 
I can now distribute my camera rtsp feed to various applications, getting over each individual camera's limitation. 
Very simple to install & working great indeed!

**Comment by aler9** [comment]
Hello, thank you very much for the appreciation, the aim is to release a software with a small number of features but with an eye on stability and efficiency. So far it has been used everywhere, from embedded computers to enterprise-level servers.

[a comment by an automated account is not shown]
